docs/adr-business-finder-approach.md

```diff
@@ -0,0 +1,46 @@
+# Decision Record: Links-based Facets for Business Finder
```

Comment:
While I agree that an ADR needs to be written, this is not the place for it. What's detailed here is the solution, and not the problem that needed to be fixed. The changes made to finder-frontend were made as a result of the proposed solution, but didn't factor into the decision.

Can you create a separate card for the ADR, and I'll get some of the people who have context of the problem to write/review it.

The documentation here best belongs in a section about how finder-frontend handles links-based facets.

Reply:
Raised https://trello.com/c/FrKLI3vc/182-write-an-adr-for-links-based-facets